R/gh_whoami.R

```diff
@@ -0,0 +1,74 @@
+#' Info on current GitHub user and token
+#'
+#' Reports wallet name, GitHub login, and GitHub URL for the current
+#' authenticated user, the first few and last characters of the token, and the
+#' associated scopes.
```

Comment:
I thought showing first 4 and last 4 characters was OK. I do the same over in googlesheets. It is helpful when you have more than one token in your life and there's no obvious way to give them names. But we can kill it or reveal even less of it.